WordPress/Classes/ViewRelated/Blog/Blog List/BlogListViewController.m

```diff
@@ -1,1073 +0,0 @@
-    [alertController addDestructiveActionWithTitle:destructiveTitle handler:^(UIAlertAction * __unused action) {
-        [self confirmRemoveSiteForIndexPath:indexPath];
-    }];
-    [self presentViewController:alertController animated:YES completion:nil];
-    [self.tableView setEditing:NO animated:YES];
-}
-
-- (void)confirmRemoveSiteForIndexPath:(NSIndexPath *)indexPath
-{
-    Blog *blog = [self.dataSource blogAtIndexPath:indexPath];
-    [self removeBlogItemsFromSpotlight:blog];
-
-    BlogService *blogService = [[BlogService alloc] initWithCoreDataStack:[ContextManager sharedInstance]];
-    [blogService removeBlog:blog];
-
-    if ([AppConfiguration isWordPress]) {
-        [ContentMigrationCoordinator.shared cleanupExportedDataIfNeeded];
-    }
-    
-    // Delete local data after removing the last site
-    if (!AccountHelper.isLoggedIn) {
-        [AccountHelper deleteAccountData];
-    }
-
-    [self.tableView reloadData];
-}
-
-- (void)hideBlogAtIndexPath:(NSIndexPath *)indexPath
-{
-    Blog *blog = [self.dataSource blogAtIndexPath:indexPath];
-    [self setVisible:NO forBlog:blog];
-    [self.tableView setEditing:NO animated:YES];
-    [self removeBlogItemsFromSpotlight:blog];
-}
-
-- (void)unhideBlogAtIndexPath:(NSIndexPath *)indexPath
-{
-    Blog *blog = [self.dataSource blogAtIndexPath:indexPath];
-    [self setVisible:YES forBlog:blog];
-    [self.tableView setEditing:NO animated:YES];
-}
-
-- (BOOL)tableView:(UITableView *)tableView shouldIndentWhileEditingRowAtIndexPath:(NSIndexPath *)indexPath
-{
-    return NO;
-}
-
-- (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSInteger)section
-{
-    // If we have more than one section, show a 2px separator unless the section has a title.
-    NSString *sectionTitle = nil;
-    if ([tableView.dataSource respondsToSelector:@selector(tableView:titleForHeaderInSection:)]) {
-        sectionTitle = [tableView.dataSource tableView:tableView titleForHeaderInSection:section];
-    }
-    if (section > 0 && [sectionTitle length] == 0) {
-        return 2;
-    }
-    return UITableViewAutomaticDimension;
-}
-
-- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath
-{
-    Blog *blog = [self.dataSource blogAtIndexPath:indexPath];
-    if (self.tableView.isEditing) {
-        UITableViewCell *cell = [tableView cellForRowAtIndexPath:indexPath];
-        UISwitch *visibleSwitch = (UISwitch *)cell.accessoryView;
-        if (visibleSwitch && [visibleSwitch isKindOfClass:[UISwitch class]]) {
-            visibleSwitch.on = !visibleSwitch.on;
-            [self setVisible:visibleSwitch.on forBlog:blog];
-        }
-        return;
-    } else {
-        blog.visible = YES;
-
-        RecentSitesService *recentSites = [RecentSitesService new];
-        [recentSites touchBlog:blog];
-
-        if (![blog isEqual:self.selectedBlog]) {
-            [[PushNotificationsManager shared] deletePendingLocalNotifications];
```

Comment:
It's likely there are other kinds of local notifications, but one use case I can think of is weekly roundup notifications for previously selected site will be unscheduled after switching site. Is this use case supported by the SwiftUI blog list?

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-iOS/pull/23455/commits/61683f4a9285ef562543fe98800373735151868f. I also found another bug in this code: it was never designed to support more than one self-hosted site 👀 

One more note: `RecentSitesService` now also appears to be unused. I'm actually not sure why it wasn't used in the new implementation and why the new `lastUsed` property was introduced in the main database. I guess we now have to live with it because this db migration already shipped 😞